qualtran/Protocols.ipynb

```diff
@@ -0,0 +1,72 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a3bea287",
+   "metadata": {},
+   "source": [
+    "# Protocols\n",
```

Comment:
If I didn't know what a protocol is, I wouldn't have understood this section. I think a simple paragraph that doesn't mention bloqs and references some good explanation should make this section more understandable.

Reply:
well I don't think we're using the word "protocol" accurately in an established way:

https://en.wikipedia.org/wiki/Protocol

but rather leaning in to its ambiguous meaning (https://en.wiktionary.org/wiki/protocol) to define it exactly how we want; i.e. below.

Reply:
To be clear is this the idiomatic meaning of protocol and not any formal CS protocol type thing

Reply:
how about this updated wording:

> We write bloqs so we can query properties, draw diagrams, and check the correctness of quantum algorithms. Basically: we write bloqs so we can do things with them! Each 'thing'--or genre of thing--we want to do has an associated "protocol". In our usage of the term, a protocol is a set of methods, functions, or procedures to structure a relevant piece of functionality.